Adds Jaeger trace data for analytics documentation

alicejw-aws opened the pull request:
Signed-off-by: alicejw <alicejw@amazon.com>

### Description
New doc for how to use Jaeger trace data for analytics.

### Issues Resolved
https://github.com/opensearch-project/documentation-website/issues/2313

### PDF for reviewers
To see the document as it will be rendered on the site, take a look at this PDF.

[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10406810/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)


### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,87 @@
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
```
This doesn't seem to render out in the rich diff?

alicejw-aws replied:
see PDF for how it will render on the site. the Liquid variables don't render in the GitHub file viewer.

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,87 @@
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the trace analytics capability. When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the D ...
+
+Jaeger trace data that you can analyze includes span data, service dependency data, and service and operation endpoint data. Jaeger span data analysis requires some configuration.
```
What is service dependency data - not sure if we have that, maybe word in a different way?

derek-ho replied:
Maybe want to add here limitation that errors need to have this flag set: --es.tags-as-fields.all=true

See this comment: https://github.com/opensearch-project/dashboards-observability/issues/83#issuecomment-1370084286

derek-ho replied:
This is also available in the product, but maybe deserves a special call out that they need to have indices in the format of *jaeger-span-* and *jaeger-service*

alicejw-aws replied:
Could you also specify which file they set this flag in?

derek-ho replied:
The flag is set as a configuration for ingestion

alicejw-aws replied:
thanks. also added the procedure you shared with me.

alicejw-aws commented:
Hi @derek-ho thanks for your review! I've updated it. Could you verify and provide tech review approval?
@anirudha , could you also please take a look and provide feedback/approval today? 
We'd like to get this into the 2.5 release if possible.

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,85 @@
+
+Jaeger indexes follow the naming conventions `jaeger-span-*` or `jaeger-service-*`.
+
+Jaeger and OTEL indexes have different field names. Therefore when you run trace analytics, you'll need to create different queries and components depending on which index type you are using.
+{:.note}
```
What does this mean? We are creating the dashboard for them?

alicejw-aws replied:
will comment this out for now. this info was in the feature brief or one of the eng tickets.

alicejw-aws replied:
i hid it with comments if we need it for 2.6 when they provide the details about those.

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,85 @@
+
+## Dashboards views
+
+After you select Jaeger for the data source, you can view all of your indexed data in the Dashboards, including Error rate, Throughput, and Services.
```
Maybe we want to separate out Service here: 
On dashboard page: 
- error rate/throughput
Trace Page
Service Page

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,85 @@
+After you select Jaeger for the data source, you can view all of your indexed data in the Dashboards, including Error rate, Throughput, and Services.
+### Error rate
+
+You can view the Trace error rate over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
```
Top 5 combinations of service and operations with the highest non-zero error **count** (we show rate separately, but it is the top 5 of count)

nitincd commented:
would suggest updating the first section to:
==
The  Trace analytics functionality in OpenSearch Observability plugin now supports Jaeger trace data. Users that use OpenSearch as the backend for their Jaeger trace data can now benefit from the analysis capabilities built into OpenSearch Trace analytics. This adds to the existing support for OTEL formatted trace data.

When you use trace analytics, you can now select from two data sources:
Data Prepper – Data ingested into OpenSearch through Data Prepper. This data source corresponds to the trace data in OpenTelemetry (OTEL) format.
Jaeger – Trace data from Jaeger stored within OpenSearch. This data source corresponds to the Jaeger format.

If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace analytics to analyze the error rates and latencies. You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
==

nitincd commented:
would suggest updating the first section to:

The  Trace analytics functionality in OpenSearch Observability plugin now supports Jaeger trace data. Users that use OpenSearch as the backend for their Jaeger trace data can now benefit from the analysis capabilities built into OpenSearch Trace analytics. This adds to the existing support for OTEL formatted trace data.

When you use trace analytics, you can now select from two data sources:
Data Prepper – Data ingested into OpenSearch through Data Prepper. This data source corresponds to the trace data in OpenTelemetry (OTEL) format.
Jaeger – Trace data from Jaeger stored within OpenSearch. This data source corresponds to the Jaeger format.

If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace analytics to analyze the error rates and latencies. You can also filter the traces and look into the span details of a trace to pinpoint any service issues.

alicejw-aws commented:
Tech review comments are now incorporated. (except need one more command from @derek-ho )
Please also see updated PDF:
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10396737/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

Thanks

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,283 @@
+
+## Step 2: View trace data in OpenSearch Dashboards
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data on http://localhost:5601
```
maybe put a link here? Example: http://localhost:5601/app/observability-dashboards#/trace_analytics/home

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,283 @@
+---
```
Will you make changes discussed yesterday of linking top level doc to this sub page?

alicejw-aws replied:
yes

anirudha reviewed (approved):
thanks

alicejw-aws commented:
all tech review comments added. 
PDF: 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10404463/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

alicejw-aws commented:
main page link: 
![Screen Shot 2023-01-12 at 9 49 49 AM](https://user-images.githubusercontent.com/88908598/212141668-e76c45b6-1e31-49b2-acdc-6583d64627b7.png)

alicejw-aws commented:
added warning not to use sample docker file in prod env. 
updated PDF: 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10404758/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+If you want to see errors in your trace data, you need to set the following ElasticSearch flag to true prior to data ingestion: `--es.tags-as-fields.all=true`.
+
+Jaeger data that is ingested for OpenSearch needs to have the  flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
```
it will not work for errors/error data will not be available for traces in trace analytics

alicejw-aws replied:
ok updated it

alicejw-aws commented:
PDFs for reviewers: 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10404916/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)
[Trace analytics- note on index page..pdf](https://github.com/opensearch-project/documentation-website/files/10404917/Trace.analytics-.note.on.index.page.pdf)

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+If you select one of the entries for Service and Operation Name and go to the **Traces** column to select a trace, it will add the service and operation as filters for you.
+
+In **Traces**, you can see the latency and errors for the filtered service and operation for each individual Trace ID in the list.
```
Traces is also a separate sub page, do we want to give it the same treatment as ###services?

alicejw-aws replied:
ok thanks

derek-ho commented on _observability-plugin/trace/index.md:
```diff
@@ -15,3 +15,9 @@ A single operation, such as a user clicking a button, can trigger an extended se
+
+## Trace Analytics with Jaeger data
+
+The Trace Analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace analytics built-in analysis capabilities.
```
Capital Analytics?

alicejw-aws replied:
it was the feature name

derek-ho commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
+
+Each time you ingest data for Jaeger, it creates a separate index for that day. The Dashboards will show the current index that has a mapping.
```
By default it creates a separate index, they can define custom index lifecycle management, does that belong here?

vagimeli replied:
Delete "The" before "Dashboards." Or, do we mean "dashboards."

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper.
+- **Jaeger** – Trace data stored within OpenSearch as its backend.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace Analytics to analyze the error rates and latencies You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
```
Lowercase "Analytics" and add period after "latencies."

alicejw-aws commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+## Data Ingestion Requirements
+
+If you want to see errors in your trace data, you need to set the following ElasticSearch flag to true prior to data ingestion: `--es.tags-as-fields.all=true`.
```
updated offline in meeting. thanks for clarification.

vagimeli replied:
Suggested rewrite to tighten wording: "To see trace data errors, set the following...."

alicejw-aws commented:
Updated PDFs after final clarification meeting and small updates. 

[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10405174/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)\
[Trace analytics-overview.pdf](https://github.com/opensearch-project/documentation-website/files/10405173/Trace.analytics-overview.pdf)

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace Analytics to analyze the error rates and latencies You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
+
+When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
```
Is OTA a proper name or an acronym? If the latter, define on first mention. Delete "the" before "Data Prepper" and "Dashboards."

Suggested rewrite for clarity: "Use the data source selector in Dashboards to indicate on which data source you want to perform trace analytics."

alicejw-aws commented:
Updated PDFs after final clarification meeting and small updates. 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10405205/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)


[Trace analytics-overview.pdf](https://github.com/opensearch-project/documentation-website/files/10405173/Trace.analytics-overview.pdf)

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
```
What kind of configuration? Should we give user an example of what they may have to configure?

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+To learn more about Jaeger data tracing, see the [Jaeger](https://www.jaegertracing.io/) open source documentation.
+
+## Data Ingestion Requirements
```
Set title in sentence case.

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+If you want to see errors in your trace data, you need to set the following ElasticSearch flag to true prior to data ingestion: `--es.tags-as-fields.all=true`.
+
+Jaeger data that is ingested for OpenSearch needs to have the  flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
```
Delete extra space between "the" and "flag."

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+Jaeger data that is ingested for OpenSearch needs to have the  flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
+
+### About Data ingestion with Jaeger indexes
```
Set title in sentence case.

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+## How to set up OpenSearch to use Jaeger data
+
+To use trace analytics with Jaeger data, you need to configure error capability for use with trace analytics. We provide a sample Docker compose file that contains the required configurations.
```
Who is "We" referring to? Suggested rewrite: "A sample Docker compose file with the required configurations is provided below."

alicejw-aws replied:
it was in a style guide that "we" refers to OpenSearch project team. 
will change wording as you suggest

hdhalter replied:
I believe we only use 'we' when referring to us, as a team, like when we say, "we recommend". I saw a good statement from Nate B. about this somewhere...

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data.
+
+Go to OpenSearch Dashboards Trace Analytics at [Trace Analytics](http://localhost:5601/app/observability-dashboards#/trace_analytics/home).
```
After the first mention of "OpenSearch Dashboards," use the shortened proper name "Dashboards" on subsequent mentions. Lowercase "Analytics" in both uses.

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze your Jaeger trace data in the Dashboards, you need to set up Trace Analytics first. To get started, see [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
```
Delete "the" before Dashboards.

Suggested rewrite for clarity: "To analyze the Jaeger trace data in Dashboards, first set up the Trace analytics functionality. See [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/)".

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+### Data sources
+
+You can specify either Data Prepper or Jaeger as your data source when you perform trace analytics.
```
We use too many personal pronouns, and it's a bit distracting. Suggestion "...as **the** data source..."

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+### Data sources
+
+You can specify either Data Prepper or Jaeger as your data source when you perform trace analytics.
+From the OpenSearch Dashboards, go to **Observability > Trace Analytics** and select Jaeger.
```
Change to read "**From Dashboards**, go to..."

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+## Dashboards views
+
+After you select Jaeger for the data source, you can view all of your indexed data in **Dashboard** view including **Error rate** and **Throughput**.
```
"**the** indexed data"

Add comma after "view"

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+### Error rate
+
+You can view the Trace error count over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
```
Lowercase "Dashboard" if you mean a dashboard in general. If we mean the "Dashboard view," then revise accordingly. Delete the comma after "Dashboard" and "also" after "and."

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+With **Throughput** selected, you can see the throughput of traces on Jaeger indexes that are coming in over time.
+
+You can select an individual Trace from **Top 5 Service and Operation Latency** list and view the detailed trace data.
```
Lowercase "Trace"

vagimeli commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,238 @@
+
+You can also see the combinations of services and operations that have the highest latency.
+
+If you select one of the entries for Service and Operation Name and go to the **Traces** column to select a trace, it will add the service and operation as filters for you.
```
"It will **automatically** add..." and then end the sentence at "filters."

vagimeli reviewed (approved):
@alicejw-aws Made some comments and suggestions. Good work!

hdhalter reviewed (commented):
Hi @alicejw-aws , can you please update the name of the PR to match our naming standards? This will appear in the release notes and should follow a similar convention. Thanks!

Naarcha-AWS commented on _observability-plugin/trace/index.md:
```diff
@@ -15,3 +15,9 @@ A single operation, such as a user clicking a button, can trigger an extended se
+
+## Trace Analytics with Jaeger data
+
+The Trace Analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace Analytics built-in analysis capabilities.
```
```suggestion
The trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace Analytics built-in analysis capabilities.
```

Naarcha-AWS commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,240 @@
+Introduced 2.5
+{: .label .label-purple }
+
+The Trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace analytics built-in analysis capabilities. This provides support for OpenTelemetry (OTEL) formatted trace data.
```
```suggestion
The trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace analytics built-in analysis capabilities. This provides support for OpenTelemetry (OTEL) formatted trace data.
```

Naarcha-AWS commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,240 @@
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper.
+- **Jaeger** – Trace data stored within OpenSearch as its backend.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace Analytics to analyze the error rates and latencies You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
```
```suggestion
If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into trace analytics to analyze the error rates and latency. You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
```

Naarcha-AWS commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,240 @@
+
+## Data Ingestion Requirements
+
+To use Trace Analytics with Jaeger data, you need to configure error capability for use with trace analytics.
```
```suggestion
To use trace analytics with Jaeger data, you need to configure error capability for use with trace analytics.
```

Naarcha-AWS commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,240 @@
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze your Jaeger trace data in the Dashboards, you need to set up Trace Analytics first. To get started, see [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
```
```suggestion
To analyze your Jaeger trace data in the Dashboards, you need to set up trace analytics first. To get started, see [Get started with trace analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
```

Naarcha-AWS commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,240 @@
+
+### Error rate
+
+You can view the Trace error count over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
```
```suggestion
You can view the trace error count over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
```

Naarcha-AWS reviewed (approved):
@natebower can verify, but I believe that "Trace Analytics" should be lowercased in most instances.

Otherwise, looks good to me.

alicejw-aws commented:
Doc review updates completed. Sending for editorial review.
Updated PDF: 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10406555/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

alicejw-aws commented:
Updated PDF after adding additional step from eng. 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10406741/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

alicejw-aws commented:
Updated PDF after adding additional step from eng. 
[Analyze Jaeger trace data - OpenSearch documentation.pdf](https://github.com/opensearch-project/documentation-website/files/10406830/Analyze.Jaeger.trace.data.-.OpenSearch.documentation.pdf)

natebower commented:
> @natebower can verify, but I believe that "Trace Analytics" should be lowercased in most instances.
> 
> Otherwise, looks good to me.

Leaving as "Trace Analytics" to match references in other documentation.

natebower commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,249 @@
+
+![Select data source]({{site.url}}{{site.baseurl}}/images/trace-analytics/service-trace-data.png)
+
+If you select an individual Trace ID, you can see more detailed information about the trace, such as time spent by the service and each span for the service and operation. You can also view the payload that you get from the index in JSON format.
```
Same comment re: "Trace ID". "time spent by the service and each span for the service and operation" is a bit too muddled/unclear. Please simplify/clarify and tag me to review. Can "that you get from" just be something like "returned by", "provided by", or "received from"?

alicejw-aws replied:
thanks! I clarified by indicating UI elements in bold font.

alicejw-aws replied:
hi @natebower, could you please verify this change? Thanks

"If you select an individual trace ID, you can see more detailed information about the trace, such as **Time spent by the service** and **Spans**. You can also view the index payload in JSON format."

natebower replied:
@alicejw-aws LGTM

natebower reviewed (commented):
@alicejw-aws Please see my comments and changes and let me know if you have any questions. Thanks!

alicejw-aws commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,249 @@
+Introduced 2.5
+{: .label .label-purple }
+
+The trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the trace analytics built-in analysis capabilities. This provides support for OpenTelemetry (OTEL) formatted trace data.
```
yes, thanks! https://opentelemetry.io/docs/

alicejw-aws commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,249 @@
+
+### Step 2: Start the cluster
+
+Run the following command to deploy the Docker compose YAML file.
```
Looks like we are going back to use colons in our style. Will make note of that. Previously when i used colons for sentences preceding code samples 
"...use the following command:" they were changed to periods.

alicejw-aws commented on _observability-plugin/trace/trace-analytics-jaeger.md:
```diff
@@ -0,0 +1,249 @@
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data.
+
+Go to Dashboards **Trace analytics** at [http://localhost:5601/app/observability-dashboards#/trace_analytics/home](http://localhost:5601/app/observability-dashboards#/trace_analytics/home).
```
yes
